test/functional/apps/dashboard/url_field_formatter.ts

```diff
@@ -48,13 +47,14 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await settings.clickIndexPatternLogstash();
       await settings.filterField(fieldName);
       await settings.openControlsByName(fieldName);
+      await settings.toggleRow('formatRow');
       await settings.setFieldFormat('url');
       await settings.controlChangeSave();
     });
 
     it('applied on dashboard', async () => {
       await common.navigateToApp('dashboard');
-      await dashboard.loadSavedDashboard('dashboard with everything');
+      await dashboard.loadSavedDashboard('dashboard with table');
```

Comment:
Looks like the test was flaky because this dashboard was flaky to load (too many panels?)
as a fix, I am adding a single-purpose dashboard as this test focuses on URL field format. This dashboard should be more stable